lib/tests/assert.sh

```diff
@@ -0,0 +1,16 @@
+# Always failing assertion with a message.
```

Comment:
`lib/tests` is for code that supports the testing of `lib` itself. Please move this file near where it's used.